assets/javascripts/utils/base64.js

```diff
@@ -46,7 +46,7 @@ define(['raven'], function (raven) {
         try {
             decoded = decodeURIComponent(escape(new AtoB()(str.replace(/-/g, '+').replace(/_/g, '/').replace(/,/g, '='))));
         } catch(e){
-            raven.Raven.captureException(e, {tags: { level: 'info' }});
```

Comment:
This is where the circular dep was, I think it could do with a comment warning that it's required by raven.